package.json

```diff
@@ -35,13 +35,13 @@
     ]
   },
   "dependencies": {
-    "can-compute": "^3.2.0",
-    "can-define": "^1.2.0",
-    "can-event": "^3.5.0",
-    "can-observation": "^3.2.0",
+    "can-compute": "canjs/can-compute#can-cid",
```

Comment:
I'm going to update this once this is resolved https://github.com/canjs/can-compute/pull/119